src/desktop/apps/order2/routes.js

```diff
@@ -50,7 +50,11 @@ export const checkoutFlow = async (req, res, next) => {
         ...res.locals,
         assetPackage: "order2",
         // header logo should link back to originating artwork
-        headerLogoHref: res.locals.sd.REFERRER,
+        // Most robust place to get artwork url from is metaphysics, so for now
+        // assign this to redirect to home and we'll do some magic in reaction
+        // to make sure it's pointing to the right place after the page has
+        // loaded
+        headerLogoHref: "/",
```

Comment:
I guess one _other_ option is to fetch the artwork object here (based on the order line item) so we can set it. It's slightly less performant because we'd be making an extra fetch, but it feels sort of wonky to be updating the header URL client-side 😅. Curious what you think!

Reply:
My preference would be to have the header defined in react, but what you suggest is a certainly less hacky than what I've done so I'll switch to that. :+1:

Reply:
Totally agree on react. Excited for when this is extracted to reaction ✨